src/ReactiveDomain/ReactiveDomain/ReactiveDomain.csproj

```diff
@@ -85,6 +88,9 @@
       <HintPath>..\..\..\packages\reactiveui-core.6.5.2\lib\Net45\ReactiveUI.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="ReactiveUI.6.5.0.0.Fakes">
```

Comment:
Should this be added to the NuGet packages file?

Reply:
not needed